airgun/views/computeresource.py

```diff
@@ -122,27 +127,43 @@ def is_displayed(self):
 class ResourceProviderDetailView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
     compprofiles = Text("//a[text()='Compute profiles']")
+    flavor = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_flavor_id')
```

Comment:
I created, for example, libvirt compute resource (I know that you can have a different one):
![image](https://user-images.githubusercontent.com/10895065/42783963-5111ebc4-8956-11e8-8a55-14748f78cbf6.png)

and it doesn't seem that I have:
```
+    availability_zone = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_availability_zone')
+    subnet = FilteredDropdown(id='s2id_compute_attribute_vm_attrs_subnet_id')
+    security_groups = MultiSelect(
+        id='ms-compute_attribute_vm_attrs_security_group_ids')
+    managed_ip = FilteredDropdown(
+        id='s2id_compute_attribute_vm_attrs_managed_ip')
```
Can you show your screen?

Reply:
It appears after clicking one the compute profile's name. Maybe it should be in another class I wasn't sure where to put it.

Reply:
I don't have infrastructure at that moment to open the page, but I see it has URL `/compute_profiles/1-1-Small/compute_resources/1-test2/compute_attributes/new`, so maybe compute profile is proper place, but still need to see a screenshot or whatever you can provide

Reply:
![screenshot from 2018-07-23 11-00-05](https://user-images.githubusercontent.com/33493796/43067442-ac2a6548-8e67-11e8-9f72-6d40cba59593.png)